### ISSUE TITLE
NVDA does not read button role information on subsequent clicks

### ISSUE THREAD

**Issue by bearaxe** [nvaccess/nvda#13737]
### Steps to reproduce:
1) go to chrome
2) browse to https://codepen.io/skaulox/pen/dydZmWL
3) click the empty button
4) observe nvda's behavior
5) click the button again
6) observe the problem

### Actual behavior:
NVDA reads out the button as a button the first time after loading the page, but does not announce the button as a button on subsequent clicks.

### Expected behavior:
When reading a button or an element with role button, we're expecting it to read out as a button every click, not just the first click.

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
release 2021.3.5
#### Windows version:
10.0.19042 Build 19042
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
n/a
#### Does the issue still occur after restarting your computer?
it happens on multiple different computers
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
n/a
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
this issue happens on the web, not in a desktop app

**Comment by XLTechie** [comment]
To save time, please install NVDA 2022.1 and test with that version.

**Comment by bearaxe** [comment]
Updated to NVDA 2022.1, retried, and issue is still present

**Comment by seanbudd** [comment]
What version of Chrome are you using?
I can't reproduce this with 2022.1

**Comment by bearaxe** [comment]
I updated Chrome to Version 102.0.5005.63 (Official Build) (64-bit) and retried, but the issue is still happening for me

**Comment by bearaxe** [comment]
Perhaps I'm misunderstanding how this should behave, and I never gave an explicit example in the initial report. When I click, NVDA says "Unmute Button, Mute, Unmute". I'm expecting it to always read either Unmute button or Mute button.  Is what I'm experiencing actually the intended behavior? Or when you say you can't reproduce, do you mean it is still reading out the button text as expected after the second+ click?

**Comment by seanbudd** [comment]
> Perhaps I'm misunderstanding how this should behave, and I never gave an explicit example in the initial report. When I click, NVDA says "Unmute Button, Mute, Unmute". I'm expecting it to always read either Unmute button or Mute button.  Is what I'm experiencing actually the intended behavior? Or when you say you can't reproduce, do you mean it is still reading out the button text as expected after the second+ click?

This is intended behaviour. There is no reason to report that the item is still a button if a user has not changed focus. It would be quite verbose otherwise.
